plugins/labhub.py

```diff
@@ -1,7 +1,105 @@
-from errbot import BotPlugin
+import json
+import os
+import re
 
+import github3
 
-class Labhub(BotPlugin):
-    """
-    GitHub and GitLab utilities
-    """
+from errbot import BotPlugin, re_botcmd
+
+
+class LabHub(BotPlugin):
+    """GitHub and GitLab utilities"""  # Ignore QuotesBear
+
+    def activate(self):
+        super().activate()
+
+        with open(os.path.join(os.environ.get('COBOT_ROOT', os.getcwd()),
+                               'plugins/LabHub.json')) as f:
+            config = f.read()
+
+        config = json.loads(config)
```

Comment:
isn't this simpler?

```python
for key, value in config.items():
    setattr(self, key, value)
```

Or just store the values right in here,  I don't see how this json file is helping much